### ISSUE TITLE
base href leading to not found errors

### ISSUE THREAD

**Issue by wondie** [LyndaExerciseFiles/angular2-essential-training#10]
Dear Justine, 

Nice job! I wanted to try the app but somehow the base href causes not found errors on many files. Please find the log here. 

```

GET http://localhost:63342/node_modules/core-js/client/shim.min.js 
localhost/:9 GET http://localhost:63342/node_modules/zone.js/dist/zone.js 
localhost/:10 GET http://localhost:63342/node_modules/reflect-metadata/Reflect.js 
localhost/:12 GET http://localhost:63342/systemjs.config.js 
localhost/:11 GET http://localhost:63342/node_modules/systemjs/dist/system.src.js 
localhost/:13 GET http://localhost:63342/resets.css 
localhost/:8 GET http://localhost:63342/node_modules/core-js/client/shim.min.js 
(index):9 GET http://localhost:63342/node_modules/zone.js/dist/zone.js 
(index):10 GET http://localhost:63342/node_modules/reflect-metadata/Reflect.js 
(index):11 GET http://localhost:63342/node_modules/systemjs/dist/system.src.js 
(index):12 GET http://localhost:63342/systemjs.config.js 
(index):13 GET http://localhost:63342/resets.css 
(index):22 Uncaught ReferenceError: System is not defined(anonymous function) @ (index):22
```

**Comment by jschwarty** [comment]
Hi Wondimagegn,

Thanks! It looks like you are running it on a different port (63342 instead of 3000) so I'm assuming you have some custom code that you have added in there. The code from the repository should be running correctly as is. I tested this both on a Windows and Mac machine and everything runs fine. Perhaps there is something in the additional code that is causing this? Can you share the changes you have? Maybe I can spot something from that.

**Comment by wondie** [comment]
Hi Justin, 

Thank you very much for the help. 
I downloaded the repository and without modifying it opened it as a project in WebStorm. I did npm install using package.json in Webstorm. I executed index.html using WebStorm. The port came from WebStorm setting. When I used `npm start` code as stated in the readme, it executed the app in port 3001 and it worked! 

So does this mean I couldn't use other servers such as apache and WebStorm's builtin server to run the app? Until the `<base href="/">` line was added, there was no such problem.
